lib/Popup/Popup.js

```diff
@@ -592,5 +592,21 @@ module.exports = kind(
 	destroy: function() {
 		this.showHideScrim(false);
 		Popup.prototype.destroy.apply(this, arguments);
-	}
+	},
+
+	/**
+	* If [spotlightModel]{@link module:moonstone/Popup~Popup#spotlightModal} property is true
+	* back key could not close popup.
+	* Because every event out-boundary of popup should be ignored when it is true.
+	*
+	* @method
+	* @public
+	*/
+	backKeyHandler: kind.inherit(function (sup) {
+		return function () {
+			if (this.spotlightModal) this.pushBackHistory();
```

Comment:
It undoes the back action. If you hit the back key (or the back button in the browser), that'll remove the popup's entry from the history stack and normally hide it (the default backKeyHandler behavior). Since it's not supposed to close though, that pushes back onto the stack so it can be removed properly in `showingChanged`.